php/psalm-baseline.xml

```diff
@@ -20,6 +20,11 @@
       <code><![CDATA[Container]]></code>
     </ClassMustBeFinal>
   </file>
+  <file src="src/Container/CommunityContainer.php">
+    <ClassMustBeFinal>
+      <code><![CDATA[CommunityContainer]]></code>
+    </ClassMustBeFinal>
```

Comment:
What would we need to do to fix this psalm issue in the code?

Reply:
okay, so maybe we should add this to the global ignore list to not spam the baseline?